README.md

```diff
@@ -24,89 +24,100 @@ The role will set up a vpn tunnel between each pair of hosts in the list of `vpn
 
 The Ansible controller requires the python `netaddr` package.
 
-## Variables
+## Global variables
```

Comment:
In general, every variable in Ansible is a global variable (but there are certain exceptions).  The convention we use for other system roles is to just use "Variables" or "Role Variables" e.g. https://github.com/linux-system-roles/template/#role-variables

Reply:
Well, we don't really having a naming convention for that - if you need to distinguish between "top level" variables and connection specific variables, I think you could use the naming "Connection-specific variables"